### ISSUE TITLE
New session about fsproj dependency manager

### ISSUE THREAD

**Issue by ThisFunctionalTom** [amplifying-fsharp/sessions#18]
### Session on fsproj dependency manager

Tomas Leko in Europe/Vienna would like to tackle:
https://github.com/dotnet/fsharp/issues/8764

#### Extra

I already did a small implementation and started adding it to the dotnet/fsproj repository. 
I would still like to refactor a little bit and add some some tests before the session. 

It would be cool if it could than be deployed with F# SDK so everyone can use it just like the nuget dependency manager.

Sadly, I do not have time on friday afternoon (Vienna/UTC). Could we do it thursday afternoon or friday morning please?

**Comment by dawedawe** [comment]
@ThisFunctionalTom A different time is definitely possible. We'll get back to you soon.

**Comment by dawedawe** [comment]
@ThisFunctionalTom How about Thursday, 13.07.  at 17:00 CEST (Vienna)? Would that work for you?

**Comment by ThisFunctionalTom** [comment]
It's perfect. Thanks. I will write it in my calendar and try to prepare as much as possible until then.

**Comment by dawedawe** [comment]
Great, we will create a session page for you to review.

A few days prior to the session we will contact you again, maybe have a short call, to  talk about some details and make sure everything is set up.

**Comment by nojaf** [comment]
Hi Tomas, I've been reading a bit on the issue.
On a practical level, what code change do you want to make and do we have a clear view of what change the team would accept in a PR?

**Comment by ThisFunctionalTom** [comment]
Hi,

there is a dependency manager for nuget which comes with dotnet fsi. There was a prototype version of an fsproj dependency manager from Chris which I forked and worked on it months ago. Sadly, it was not integrated in the fsharp repo and is not installed with dotnet installation and I would like to change it.

As I was trying to update the code to work with .NET 7 I changed the implementation so that it does not have any dependencies on other nuget packages.  Here is the branch: https://github.com/ThisFunctionalTom/fsharp/tree/DependencyManagerFsProj

I would like to do following things with you:
- maybe shortly say about what happened till now
- I would like to discuss with you the current implementation
- currently I have a problem/bug. If I do not find the solution to my problem until the session maybe we could look into it together
- I will create a PR and would like to prepare it with you so that it can be merged back.

The goal would be to prepare the PR so that it could be merged into dotnet/fsharp. Let's see if I am only dreaming or if it is possible 😀.
